src/components/Tools.js

```diff
@@ -27,6 +27,26 @@ class Tools extends Component{
               <Button variant="primary">Portfolios</Button>
             </Card.Body>
           </Card>
+          <Card style={{ width: '18rem' }}>
+            <Card.Img variant="top" src="holder.js/100px180" />
```

Comment:
The "holder.js/100px180" shows up as a broken image for me.  If you have an image that displays there you may have forgotten to push it to Github.